### PR TITLE
Extend TextRender with make_contents postprocessing (and minor cleanup)

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

**Comment by stdweird** [comment]
This replaces the CAF part of #85

**Comment by jrha** [comment]
LGTM

File: src/main/perl/TextRender.pm [hunk 1]
```diff
@@ -465,12 +539,26 @@ sub render_json
     return $j->encode($self->{contents});
 }
 
+# search and replace the YAML boolean PREFIX
+# private function, call diretcly for testing only
```
Comment (jrha) [inline src/main/perl/TextRender.pm]: diretcly

File: src/main/perl/TextRender.pm [hunk 2]
```diff
@@ -465,12 +539,26 @@ sub render_json
...
+sub _yaml_replace_boolean_prefix
+{
+    my ($self, $yamltxt) = @_;
+    # Implicit quoting could be enabled in the YAML::XS Dump.
+    $yamltxt =~ s/('|")?$YAML_BOOL_PREFIX(true|false)\1?/$2/g;
+    if ($yamltxt =~ m/$YAML_BOOL_PREFIX/) {
+        # just in case
```
Comment (jrha) [inline src/main/perl/TextRender.pm]: in case what?

File: src/test/perl/textrender.t [hunk 3]
```diff
@@ -374,8 +401,36 @@ name_level0: value_level0
...
 ok($trd->load_module('YAML::XS'), "YAML::XS loaded");
+ok(! $trd->{method_is_tt}, "method_is_tt false for yaml");
 is("$trd", $res, "yaml module rendered correctly");
 
+
+# true/false tests
+$trd = CAF::TextRender->new('yaml', $YAML_BOOL, eol=>0);
+my $txt = "$trd";
+$txt =~ s/\s//g;
+is($txt, '---no:falseyes:true',
```
Comment (jrha) [inline src/test/perl/textrender.t]: Wait... what?
falseyes?

Comment (stdweird) [inline src/test/perl/textrender.t]: all whitespace is stripped; this is 
```yaml
no: false
yes: true
```

**Comment by stdweird** [comment]
@jrha updated remarks

File: src/test/perl/textrender.t (hunk 3 above)
Comment (jrha) [inline src/test/perl/textrender.t]: Ah I see :relieved:

**Pull request merged**